frontend/public/components/nav/perspective-nav.tsx

```diff
@@ -111,12 +111,12 @@ const PerspectiveNav: React.FC<{}> = () => {
           const { id, name } = item.properties;
           return <NavSection id={id} title={name} key={id} isGrouped={!name} />;
         }
-        if (isNavItem(item)) {
-          return createLink(item, true);
-        }
         if (isSeparatorNavItem(item)) {
           return <NavItemSeparator key={`separator-${index}`} />;
         }
+        if (isNavItem(item)) {
+          return createLink(item, true);
+        }
```

Comment:
@christianvogt's suggestion was to remove the check for `isNavItem` and directly return `createLink` in the end since `isNavItem` would be true for all nav extensions.

```suggestion
        if (isNavItem(item)) {
          return createLink(item, true);
        }
```